packages/components/src/__overlays__/Tooltip/v3/_docs/Tooltip.stories.tsx

```diff
@@ -32,57 +31,27 @@ export default meta
 type Story = StoryObj<typeof meta>
 
 export const OnButton: Story = {
-  render: ({ defaultOpen, isOpen, ...args }) => (
-    <TooltipTrigger defaultOpen={defaultOpen} isOpen={isOpen}>
+  render: args => (
```

Comment:
this looks like git conflict resolved the other way. The red is what we want in this file

Reply:
Ah nice catch, will clear that one up 👍

Reply:
Reverted back to the original state, should be good to merge if all is hunky dory